src/Slider.js

```diff
@@ -5,18 +5,14 @@ import mdlUpgrade from './utils/mdlUpgrade';
 class Slider extends React.Component {
     static propTypes = {
         className: PropTypes.string,
-        max: PropTypes.number.isRequired,
-        min: PropTypes.number.isRequired,
-        onChange: PropTypes.func.isRequired,
+        max: PropTypes.number,
```

Comment:
Why it's not required anymore?
Creating a Slider without the min/max doesn't make any sense